utils/wasm-instrument/src/syscalls.rs

```diff
@@ -544,6 +624,18 @@ impl PtrInfo {
     }
 }
 
+/// Hash type.
+///
+/// Used to distinguish between different hash types in the syscall signatures.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub enum HashType {
+    ActorId,
+    CodeId,
+    MessageId,
+    ReservationId,
+    SubjectId,
```

Comment:
I'd specify here that it's used for `gr_random` as meaning of `SubjectId` is not clear from the context

Reply:
done in 0c0eb6318d0d54606150ee7b65549ec7f4accb00